libs/wingsdk/src/target-sim/event-mapping.ts

```diff
@@ -0,0 +1,77 @@
+import { Construct } from "constructs";
+import { ISimulatorResource } from "./resource";
+import {
+  EVENT_MAPPING_TYPE,
+  EventMappingSchema,
+  EventSubscription,
+  FunctionHandle,
+} from "./schema-resources";
+import { simulatorHandleToken } from "./tokens";
+import { bindSimulatorResource, makeSimulatorJsClient } from "./util";
+import { fqnForType } from "../constants";
+import * as core from "../core";
+import { Resource, App } from "../core";
+import {
+  BaseResourceSchema,
+  ISimulatorResourceInstance,
+} from "../testing/simulator";
+
+export interface IEventPublisher extends ISimulatorResourceInstance {
+  addEventSubscription: (
+    subscriber: FunctionHandle,
+    subscriptionProps: EventSubscription
+  ) => Promise<void>;
+}
+
+export const EVENT_MAP_FQN = fqnForType("sim.EventMapping");
+
+export interface EventMappingProps {
+  subscriber: core.IResource;
+  publisher: core.IResource;
+  subscriptionProps: EventSubscription;
+}
+
+export class EventMapping extends Resource implements ISimulatorResource {
```

Comment:
Docstring

@eladb It looks like one side effect of #2076 is since the target-specific modules aren't exported by JSII, our API checker no longer catches docstrings missing in them - thoughts?